lucene/backward-codecs/src/test/org/apache/lucene/index/TestBackwardsCompatibility.java

```diff
@@ -1834,33 +1834,6 @@ private void searchExampleIndex(DirectoryReader reader) throws IOException {
     assertTrue(firstDate <= lastDate);
   }
 
-  /**
-   * Tests that {@link CheckIndex} can detect invalid sort on sorted indices created
-   * before https://issues.apache.org/jira/browse/LUCENE-8592.
-   */
-  public void testSortedIndexWithInvalidSort() throws Exception {
```

Comment:
This test now failed with the `CheckIndex` changes. Since it [was removed in `main` a few months ago](https://github.com/apache/lucene/commit/2afa6cb00a2933a15e65d714b3c564c314059466#diff-cb558a4f36adaa7a363b310a5ea0aa2afaf12b0177862ee36312007eeab4c08dL1729) but not backported, I temporarily removed it from here as well. However, I doubt this is the right way of handling this test failure, and suspect there might be a previous change in `CheckIndex` that was not compatible with this test and should be taken out as well. I'll look into this a bit more in the next few days.